scripts/asm-installer/asm_vm

```diff
@@ -987,13 +988,17 @@ retrieve_workload_asm_revision() {
   WORKLOAD_ASM_REVISION="$(retry 2 kubectl get namespace "${WORKLOAD_NAMESPACE}" -ojson \
     | jq -r '.metadata.labels["'${ASM_REVISION_LABEL_KEY}'"]')"
   if [[ -z "${WORKLOAD_ASM_REVISION}" ]]; then
-    { read -r -d '' MSG; fatal "${MSG}"; } <<EOF || true
+      WORKLOAD_ASM_REVISION=default
```

Comment:
Do we have `istio-default` svc in the CI cluster? If not, we would either need to change the script to allow the `WORKLOAD_ASM_REVISION` be empty string or change service proxy agent code to have a different logic for `default` revision.